examples/compose/docker-compose.beginners.yml

```diff
@@ -285,10 +285,12 @@ services:
       - vtctld
       - set_keyspace_durability_policy
     ports:
+      - "13200:8080"
       - "13000:3000"
     volumes:
       - ".:/script"
     environment:
+      - WEB_PORT=8080
```

Comment:
I have fixed the docker-compose examples to also run the vtorc debug UI. This fail on `main` until the new vitess-lite image is out. I have manually tested that the example works with the new image that I built locally.